api/schemas/AppConnection.yaml

```diff
@@ -0,0 +1,13 @@
+type: object
+description: Connections between users and organizations
```

Comment:
Perchè connections (plurale)? Un'istanza di questo schema rappresenta una connessione. Inoltre, più che essere una connessione _fra_ utente _e_ organizzazione, è una connessione _di_ un utente _ad_ un'organizzazione

Reply:
concordo. l'avevo già rinominato in locale :D